input/fsh/codeSystem/codeSystem.fsh

```diff
@@ -0,0 +1,1319 @@
+* #9860137 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860137 ^property[0].code = #conceptclass
+* #9860137 ^property[=].valueString = "Misc"
+* #9860137 ^property[+].code = #datatype
+* #9860137 ^property[=].valueString = "N/A"
+* #discontinuation_date "Discontinuation date"
+* #discontinuation_date ^designation[0].language = #en
+* #discontinuation_date ^designation[=].value = "Discontinuation date"
+* #discontinuation_date ^designation[=].use.code = #Fully-Specified
+* #discontinuation_date ^property[0].code = #conceptclass
+* #discontinuation_date ^property[=].valueString = "Misc"
+* #discontinuation_date ^property[+].code = #datatype
+* #discontinuation_date ^property[=].valueString = "Date"
+* #9860143 "Treatment end date"
+* #9860143 ^designation[0].language = #en
+* #9860143 ^designation[=].value = "Date treatment stopped"
+* #9860143 ^designation[+].language = #en
+* #9860143 ^designation[=].value = "Treatment end date"
+* #9860143 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860143 ^designation[+].language = #es
+* #9860143 ^designation[=].value = "Fecha en la que se suspendió el tratamiento"
+* #9860143 ^property[0].code = #conceptclass
+* #9860143 ^property[=].valueString = "Question"
+* #9860143 ^property[+].code = #datatype
+* #9860143 ^property[=].valueString = "Date"
+* #9860145 "Transferred out" "Patient transferred out to other services"
+* #9860145 ^designation[0].language = #en
+* #9860145 ^designation[=].value = "Transferred out"
+* #9860145 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860145 ^designation[+].language = #fr
+* #9860145 ^designation[=].value = "Transférer au"
+* #9860145 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860145 ^designation[+].language = #fr
+* #9860145 ^designation[=].value = "Reféré vers"
+* #9860145 ^designation[+].language = #ht
+* #9860145 ^designation[=].value = "Transfere deyò"
+* #9860145 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860145 ^property[0].code = #conceptclass
+* #9860145 ^property[=].valueString = "Misc"
+* #9860145 ^property[+].code = #datatype
+* #9860145 ^property[=].valueString = "N/A"
+* #9860147 "Died"
+* #9860147 ^designation[0].language = #en
+* #9860147 ^designation[=].value = "Died"
+* #9860147 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860147 ^designation[+].language = #es
+* #9860147 ^designation[=].value = "Muerte"
+* #9860147 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860147 ^designation[+].language = #fr
+* #9860147 ^designation[=].value = "Dècédé\t"
+* #9860147 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860147 ^designation[+].language = #ht
+* #9860147 ^designation[=].value = "mouri"
+* #9860147 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860147 ^property[0].code = #conceptclass
+* #9860147 ^property[=].valueString = "State"
+* #9860147 ^property[+].code = #datatype
+* #9860147 ^property[=].valueString = "N/A"
+* #9860149 "Lost to followup" "Patients that are lost to followup.. boolean question."
```

Comment:
CIEL has a code for this. Do we have specific reasons for choosing to define new codes as opposed to using existing CIEL codes? See concept 5240 - https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5240/

Reply:
@jamlung-ri , This is not a new code that I created. It already exist in CIEL with concept ID "9860149 " , Please confirm.

Reply:
Could you share a link to that CIEL concept? I am unable to find it on OCL Online when searching for that code or concept name: https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5240/?q=lost%20to%20followup&isTable=true&isList=false&page=1

Reply:
@jamlung-ri , My bad. You are right this was a custom code I created. I have removed it and replaced with the concept Id 5240.

Reply:
@jamlung-ri , I have tried adding this concept in my CIEL sources and its not going through. Please check.